test/everypolitician_test.rb

```diff
@@ -59,6 +59,15 @@ def test_country_and_legislature_convenience_method
     end
   end
 
+  def test_sources_dir_convenience_method
+    VCR.use_cassette('countries_json') do
+      legislature = Everypolitician::Legislature::from_sources_dir('Australia/Senate')
```

Comment:
This should probably use a `.` rather than `::` between the class name and the method name.